pontoon/projects/templates/projects/widgets/project_list.html

```diff
@@ -63,8 +63,9 @@ <h4>
   </table>
   {% if request %}
     <div id="request-item-note">
-      <p>The requested projects won’t be enabled automatically: an email will be sent to Pontoon admins and team managers to evaluate the request. We’ll try to respond as soon as possible. Please make sure to get in touch with the <a href="{{ url('pontoon.teams.info', locale.code) }}">localization team</a> if you are a new contributor.</p>
+      <p class="locale-projects">The requested projects won’t be enabled automatically: an email will be sent to Pontoon admins and team managers to evaluate the request. We’ll try to respond as soon as possible. Please make sure to get in touch with the <a href="{{ url('pontoon.teams.info', locale.code) }}">localization team</a> if you are a new contributor.</p>
+      <p class="pretranslation">Pretranslation won’t be enabled automatically for the requested projects: an email will be sent to Pontoon admins and team managers to evaluate the request. We’ll try to respond as soon as possible. Note that, if a locale is not using pretranslation yet, this will require additional time to train a custom translation engine.</p>
```

Comment:
I added "We’ll try to respond as soon as possible.", which we also use in the project request description.